be/src/vec/olap/olap_data_convertor.h

```diff
@@ -422,6 +422,7 @@ class OlapBlockDataConvertor {
         OlapColumnDataConvertorBaseUPtr _key_convertor;
         OlapColumnDataConvertorBaseUPtr _value_convertor;
         std::vector<const void*> _results;
+        PaddedPODArray<UInt64> _offsets;
```

Comment:
it may be not safe to return address of _offsets in _results since the convertor lifecycle may be out of the return results' lifecycle